### ISSUE TITLE
[Bug]: Unable to install on Windows server 2022

### ISSUE THREAD

**Issue by chottuthejimmy** [wavetermdev/waveterm#1373]
### Current Behavior

Getting this error when I try to open 

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'string_decoder' imported from 
C:\Users\testUser\AppData\Local\Programs\awstem\resources\app.asar\dist\main\index.js
    at packageResolve (node:internal/modules/esm/resolve:864:9)
    at moduleResolve (node:internal/modules/esm/resolve:913:11)
    at defaultResolve (node:internal/modules/esm/resolve:1118:11)
    at ModuleResolve (node:internal/modules/esm/resolve:542:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:510:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:250:39)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:96:40)
    at link (node:internal/modules/esm/module_job:36:36)
```

### Expected Behavior

Open the GUI

### Steps To Reproduce

1. the steps are in the recordings attached

### Wave Version

latest

### Platform

Windows

### OS Version/Distribution

Windows server 2022

### Architecture

x64

### Anything else?

here's the [dashcam rec](https://app.dashcam.io/replay/674e5a83562a0221cf215844?share=2SpUqShYPV0EXJHnP9VSBg)

### Questionnaire

- [ ] I'm interested in fixing this myself but don't know where to start
- [ ] I would like to fix and I have a solution
- [ ] I don't have time to fix this right now, but maybe later

**Comment by esimkowitz** [comment]
Found the issue, there was a bug in Vite 6 that was patched in 6.0.1: https://github.com/vitejs/vite/issues/18810

**Comment by chottuthejimmy** [comment]
Nicee. 
So u think testdriver is helpful ?
@esimkowitz would mean the whole world to us if at all if you could mention us on any social media that you use. 
Just share your experience so far.
